app/views/index/global.phtml

```diff
@@ -46,7 +46,8 @@
 					$url_base['params']['get'] = 'f_' . $feed->id();
 			?>
 			<li id="f_<?= $feed->id() ?>" class="item feed<?= $error, $empty, $feed->mute() ? ' mute' : '' ?>"
-				data-unread="<?= $feed->nbNotRead() ?>" data-priority="<?= $feed->priority() ?>">
+				data-unread="<?= $feed->nbNotRead() ?>" data-priority="<?= $feed->priority() ?>"
+				title="<?= strlen($error) > 0 ? _t('sub.feed.error') : ''?> <?= strlen($empty) > 0 ? _t('sub.feed.empty') : ''	?>">
```

Comment:
I think `$error === ''` might be slightly faster than `strlen()` and it makes a bit more sense to me